web/src/lib/static/loadDisplayContent.ts

```diff
@@ -1,50 +1,57 @@
-import { ProfileDisplayContent, RoadmapDisplayContent, TasksDisplayContent } from "@/lib/types/types";
+import {
+  emptyOwningFlowContent,
+  emptyProfileContent,
+  emptyStartingFlowContent,
+  IndustryFieldContent,
+  LegalFieldContent,
+  LoadDisplayContent,
+  OwningFlowContent,
+  owningFlowDisplayFields,
+  ProfileContent,
+  profileDisplayFields,
+  RadioFieldContent,
+  RoadmapDisplayContent,
+  startFlowDisplayFields,
+  StartingFlowContent,
+  TasksDisplayContent,
+  TextFieldContent,
+  UserContentType,
+} from "@/lib/types/types";
 import { getMarkdown } from "@/lib/utils/markdownReader";
 import { LegalStructures } from "@businessnjgovnavigator/shared";
 import fs from "fs";
 import path from "path";
 
 const displayContentDir = path.join(process.cwd(), "..", "content", "src", "display-content");
 
-export const loadProfileDisplayContent = (): ProfileDisplayContent => {
-  const loadFile = (filename: string): string =>
-    fs.readFileSync(path.join(displayContentDir, "onboarding", filename), "utf8");
-
-  const hasExistingBusiness = getMarkdown(loadFile("has-existing-business.md"));
-  const businessName = getMarkdown(loadFile("business-name.md"));
-  const industry = getMarkdown(loadFile("industry.md"));
-  const legalStructure = getMarkdown(loadFile("legal-structure.md"));
-  const municipality = getMarkdown(loadFile("municipality.md"));
-  const specificHomeContractor = getMarkdown(loadFile("industry-home-contractor.md"));
-  const specificEmploymentAgency = getMarkdown(loadFile("industry-employment-agency.md"));
-  const specificLiquor = getMarkdown(loadFile("industry-liquor.md"));
-  const specificHomeBased = getMarkdown(loadFile("municipality-home-based-business.md"));
-  const employerId = getMarkdown(loadFile("employer-id.md"));
-  const entityId = getMarkdown(loadFile("entity-id.md"));
-  const notes = getMarkdown(loadFile("notes.md"));
-  const taxId = getMarkdown(loadFile("tax-id.md"));
-  const certifications = getMarkdown(loadFile("certifications.md"));
-  const existingEmployees = getMarkdown(loadFile("existing-employees.md"));
+export const loadUserDisplayContent = (): LoadDisplayContent => {
+  const loadFile = (filename: string, type: UserContentType): string =>
+    fs.readFileSync(path.join(displayContentDir, "onboarding", type.toLowerCase(), filename), "utf8");
```

Comment:
I'd prefer if we had an explicit map / Record object to map from the type "STARTING" to the folder name "starting" rather than just `toLowerCase`.

It's one of those subtle things where, if it's not explicit, it's hard to know that the folder name and the type name are intrinsically connected, and you might not realize that months from now if someone wants to change it, and wouldn't know to change this line here. Whereas an explicit map will decouple that intrinsic linkage and make it visible.